Adopters.md

```diff
@@ -0,0 +1,21 @@
+# OpenEBS LVM-LocalPV Adopters
+
+This is the list of organizations and users that publicly shared details of how they are using OpenEBS LVM-LocalPV CSI driver for running their Stateful workloads. Please send PRs to add or remove organizations/users.
+
+[click here](https://github.com/openebs/openebs/issues/2719) to see the list of organizations/users who have publicly shared the usage of OpenEBS.
+
+The list of organizations that have publicly shared the usage of LVM-LocalPV.
```

Comment:
let us remove this for now as there is no adopters. Will add this section later.